dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -625,12 +633,21 @@ def decrypt_v10(encrypted_password: bytes) -> str:
 
 
 def decrypt_v10_2(encrypted_password: bytes, key: bytes) -> str:
-    """
-    struct chrome_pass {
-        byte signature[3] = 'v10';
-        byte iv[12];
-        byte ciphertext[EOF];
-    }
+    """Decrypt a version 10 type 2 password.
+
+    Args:
+        encrypted_password (bytes)
+        key (bytes)
+
+    Returns: decrypted password string
+
+    .. code-block::
+
+        struct chrome_pass {
+            byte signature[3] = 'v10';
+            byte iv[12];
+            byte ciphertext[EOF];
+        }
```

Comment:
```suggestion
    .. code-block::

        struct chrome_pass {
            byte signature[3] = 'v10';
            byte iv[12];
            byte ciphertext[EOF];
        }

    Args:
        encrypted_password: The encrypted password bytes.
        key: The encryption key.

    Returns:
        Decrypted password string.
```
Is there maybe a `Reference:` thing you can use for the struct?